File: Modul-2/Soal-4/PRAK204-2410817220012-NOOR-KHALISA.py
r = float(input("Jari-jari: "))
t = float(input("Tinggi bejana: "))
pi = 22/7

volume = pi * r * r * t
luas = 2 * pi * r * (r + t)
keliling = 2 * pi * r

print(f"Volume: {volume:.2f}")
print(f"Luas: {luas:.2f}")
print(f"Keliling: {keliling:.2f}")

File: Modul-2/Soal-4/PRAK204-2410817220012-NOOR-KHALISA.c
#include <stdio.h>

int main(){
    float r, t, volume, luas, keliling;
    const float pi = 22.00 / 7.00;
    printf("Jari-jari: ");
    scanf("%f", &r);
    printf("Tinggi bejana: ");
    scanf("%f", &t);

    volume = pi * r * r * t;
    luas = 2 * pi * r * (r + t);
    keliling = (2 * pi * r);

    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}